src/runtime.js

```diff
@@ -8,14 +8,14 @@ import Variable, {TYPE_IMPLICIT, no_observer} from "./variable";
 export var variable_invalidation = {};
 export var variable_visibility = {};
 
-export default function Runtime(builtins, globals) {
+export default function Runtime(builtins, global = noop) {
```

Comment:
Looking great, but we should ship the `notebook-runtime` with a default `global` function that looks up the value from `window`, instead of a noop, right?

... Or are you looking into the future for notebook-runtime-on-Node.js support?

Reply:
Okay, switched to window_global as the default, and fixed all of the instances in our tests where we were calling *module*.variable(*selector*) (which is no longer supported since we extracted the observer/inspector) to instead call *module*.variable(true).